src/include/storage/data_table.h

```diff
@@ -302,8 +309,8 @@ class DataTable {
   bool SelectIntoBuffer(common::ManagedPointer<transaction::TransactionContext> txn, TupleSlot slot,
                         RowType *out_buffer) const;
 
-  void InsertInto(common::ManagedPointer<transaction::TransactionContext> txn, const ProjectedRow &redo,
-                  TupleSlot dest);
+  void (*InsertInto)(common::ManagedPointer<transaction::TransactionContext> txn, const ProjectedRow &redo,
+                     TupleSlot dest);
```

Comment:
This is a style point for discussion.  Clang-Tidy treats this as a member rather than a method.  In this case it is private and not much of an issue, but I think in general we should keep function pointers which will be used as methods named with the same convention as regular methods.